### ISSUE TITLE
Double-duplicate breaks file list

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/ios-app#182]
### Steps to reproduce
1. Duplicate `Image.jpg` ==> `Image copy.jpg`
2. Duplicate `Image.jpg` again ==> no additional file ??
3. Open `Image copy.jpg` ==> download doesn't start
4. Delete `Image copy.jpg` ==> `Image copy.jpg` gone, but progress bar stays forever in broken state
5. Pull-to-refresh ==> `Image copy.jpg` is back again
6. Open any other file ==> download doesn't start

**Workaround:**
Delete account and re-create again

### Expected behaviour
Double-duplicate should create another copy

### Actual behaviour
Double-duplicate breaks some features of the app

### Client
**iOS version:**
iOS 12.1 (16B92)

**ownCloud app version:**
ownCloud beta version 1.0 build 85 (https://github.com/owncloud/ios-app/commit/5df9764c4975f28c21eb7cb24eb133f67e6b7a59) 

**Device model:**
Model: iPhone XS Max

### Server configuration
**ownCloud version:**
```
$ curl https://demo.owncloud.com/status.php
{"installed":true,"maintenance":false,"needsDbUpgrade":false,"version":"10.0.10.4","versionstring":"10.0.10","edition":"Enterprise","productname":"ownCloud"
```

@felix-schwarz @pablocarmu

**Comment by felix-schwarz** [comment]
Fixed as of e1263dae4cc2c7b458fcc281e3da5ce7be65269b.

**Comment by jesmrec** [comment]
Fixed in #200
